### DISCUSSION TITLE
CLI Plans

### DISCUSSION THREAD

**Discussion by EvolutionX-10** [sern-handler/cli#15] [Announcements]
Here are our current plans for [CLI](https://github.com/sern-handler/cli), there are no ETAs for them and can be changed anytime!

# User Interface

- [ ] Make the error/warning messages clearer
- [ ] More colors in CLI

# CLI Commands

- [x] init
    - [x] init -y
- [ ] command
- [x] plugin
- [x] extra

# Templates

- [x] [TypeScript](https://typescriptlang.org/) template
    - [x] CJS
    - [ ] ESM
- [x] [JavaScript](https://www.youtube.com/watch?v=j7gKwxRe7MQ) template
    - [ ] ESM
    - [x] CJS

**Comment by Murtatrxx** [comment]


**Reply by EvolutionX-10** [reply]
hmm, will do if @jacoobes approves, else it'll stay here

**Reply by jacoobes** [reply]
roadmap will be more general, keep roadmap for the big picture, cli todo right here is better
